plugins/aws/kms/kmsKeyPolicy.js

```diff
@@ -97,6 +106,24 @@ module.exports = {
             }
 
             async.each(listKeys.data, function(kmsKey, kcb){
+                if (config.kms_ignore_aws_managed_keys) {
+                    var describeKey = helpers.addSource(cache, source,
+                        ['kms', 'describeKey', region, kmsKey.KeyId]);
+                
+                    if (!describeKey || describeKey.err || !describeKey.data || !describeKey.data.KeyMetadata) {
+                        helpers.addResult(results, 3, `Unable to query for KMS Key: ${helpers.addError(describeKey)}`, region);
+                        return kcb();
+                    }
+    
+                    let keyLevel = helpers.getEncryptionLevel(describeKey.data.KeyMetadata, helpers.ENCRYPTION_LEVELS);
+    
+                    if (keyLevel == 2) {
```

Comment:
where are you checking if its aws managed or not?

Reply:
If its aws-managed, level will be 2